### DISCUSSION TITLE
Dark mode for nextjs docs

### DISCUSSION THREAD

**Discussion by Aakash447** [vercel/next.js#29026] [Ideas]
As vscode is in dark mode default it would be better if we get nextjs docs in dark mode also. I am using extension currently but it can't copy the code on website so I have to toggle the extension to off then copy code then again toggle the extension to dark mode. It will be better if all web goes in dark mode with toggle option of light mode importantly for docs and top websites which are faced by millions of people.

**Comment by wbt** [comment]
See also: https://github.com/vercel/next.js/issues/41792
